Sources/QuickTests/Helpers/XCTestObservationCenter+QCKSuspendObservation.m

```diff
@@ -0,0 +1,23 @@
+@import XCTest;
+#import <objc/runtime.h>
+
+@interface XCTestObservationCenter (Redeclaration)
+- (NSMutableSet *)observers;
+@end
+
+@implementation XCTestObservationCenter (QCKSuspendObservation)
+
+static BOOL (^isFromApple)(id, BOOL *) = ^BOOL(id observer, BOOL *stop){
+    return [[NSBundle bundleForClass:[observer class]].bundleIdentifier containsString:@"com.apple.dt.XCTest"];
```

Comment:
nit-pick: As a maintainer I would love an inline comment here describing why this `isApple` predicate is necessary. I understand I can `git blame` and find this pull request, but \*hand waves\* code locality something something.